index.html

```diff
@@ -320,16 +344,39 @@ <h3> Normative Specifications </h3>
               </p>
 
             </dd>
+            <dt> <a href="https://webscreens.github.io/window-placement/">Multi-Screen Window Placement</a></dt>
+            <dd>
+              <p>
+                An API that allows a web application to query its device for
+                information about directly connected displays, and additional APIs
+                to position windows relative to those displays.  Networked
+                presentation displays are out of scope for this API.</p>
+              <p class="draft-status"> <b>Draft state:</b> Editor's Draft</p>
+              <p>
+                This specification was incubated in the Second Screen Community
+                Group from Oct 2019 to Sep 2021, and was adopted by the Second
+                Screen Working Group as a deliverable in Sep 2021.  The Working
+                Group will publish the specification as an Editor's Draft after
+                rechartering is complete.
```

Comment:
Technically, the spec is incubated up until its adoption by the WG, so I guess we could say:

```suggestion
                This specification was incubated in the Second Screen Community
                Group from Oct 2019.
```

We can probably leave out the "as an Editor's Draft" part and defer to the W3C Process (linked from the "About this Charter" section) for the details of the Rec Track.

Reply:
Especially since an "Editor's Draft" has no official standing so publishing a spec as an Editor's Draft does not really mean anything